fixtures/common/exercises/track-and-field/metadata.yml

```diff
@@ -0,0 +1,3 @@
+---
+blurb: "This is a track hint."
+source: "The internet."
```

Comment:
I'm fairly sure we don't need to add an extra exercise to test this.
(please convince me if we do.)